doc/lua_api.txt

```diff
@@ -3261,6 +3261,14 @@ Player Inventory lists
 * `hand`: list containing an override for the empty hand
     * Is not created automatically, use `InvRef:set_size`
     * Is only used to enhance the empty hand's tool capabilities
+* `offhand`: list containing the offhand wielded item.
+    * Is not created automatically, use `InvRef:set_size`
+    * Will be used for placements and secondary uses if the
+        main hand does not have any node_place_prediction, on_place
+        or on_secondary_use callbacks.
+    * Is passed to on_place and on_secondary_use callbacks; make sure
+        mods are aware of the itemstack not neccessarily being
+        located in the main hand.
```

Comment:
quick question: how do mods know which hand the item is in?

I know this is not needed normally (return value will be set), but how does it work otherwise?

Reply:
not at all right now. should i add an argument to the callback?

Reply:
I think it'd be good to implement a `set_offhand_wielded_item` for players and then add a boolean to the relevant callbacks